README.md

```diff
@@ -1,2 +1,201 @@
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <h3><b>Captone Project : Hair Styles </b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+  - [Live Demo and Video](#demo)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+# 📖 Captone Project : Hair Styles  <a name="about-project"></a>
+
+**The React capstone project is an adaptive web application that showcases various hairstyles and corresponding hair details sourced from an open API. Developed using React, Redux, HTML, and CSS, this project ensures a responsive user experience.**
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+The various stacks used for the project includes the following
+<details>
+  <summary>React</summary>
+</details>
+
+<details>
+  <summary>HTML</summary>
+</details>
+
+<details>
+  <summary>CSS</summary>
+</details>
+
+<details>
+<summary>Linters</summary>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+
+- **Incorporate React components.**
+- **Implement React props.**
+- **Integrate React Router for navigation.**
+- **Establish a connection between React and Redux.**
+- **Manage events within a React application.**
+- **Compose integration tests using a React testing library.**
+- **Apply styles in a React application.**
+- **Employ React lifecycle methods effectively.**
+- **Adhere to React best practices and language style guides in your code.**
+- **Utilize store, actions, and reducers in React.**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To clone and run this application, you'll need [Git](https://git-scm.com) and [Node.js](https://nodejs.org/en/download/) (which comes with [npm](http://npmjs.com)) installed on your computer.
+
+### Prerequisites
+From your command line:
+
+On macOS, by using [Homebrew](https://brew.sh/) you can install.
+
+```bash
+# install git
+$ brew install git
+```
+
+```bash
+# install node
+$ brew install node
+```
+
+On Windows, by using [chocolatey](https://chocolatey.org) you can install.
+
+```bash
+# install git
+C:\> choco install git
+```
+
+
+```bash
+# install node
+C:\> choco install node
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```bash
+# Clone this repository
+$ git clone https://github.com/yemidada/bootstore.git
+
+# Go into the repository
+$ cd bootstore
+
+# Install dependencies
+$ npm install
+
+```
```

Comment:
- [ ] Let's have the correct setup description of our project so it's clearly documented should one needs to clone the project. Currently, you're having descriptions relating to the bookstore project and not the actual capstone project. Kindly modify this section and use the real repo to be cloned should one wants to. 👍🏽